### ISSUE TITLE
allow no-invalid-link-text to be configured for custom link component names

### ISSUE THREAD

**Issue by fivetanley** [ember-template-lint/ember-template-lint#2969]
Right now, the `no-invalid-link-text` rule checks for invalid link text on `<LinkTo>` and `<a>` tags. It would be great if we could allow a custom option to the config for tag names, so that developers using custom/wrapper components could specify additional component invocations to check.

For example, say a developer has an `ExternalLink` component that they use to consistently style links leading away from the application. We would still want those links to have more meaningful text than "learn more"/etc.

Right now, the rule passes for the following:

```handlebars
<ExternalLink @href={{this.externalLink}}>Learn more</ExternalLink>
```

We would want the above code to fail the `no-invalid-link-text` check.

The config could look something like this:

```js
// .template-lintrc.js

module.exports = {
  extends: 'recommended',
  rules: {
    'no-invalid-link-text': {
      additionalTags: ['ExternalLink']
    }
  }
};
